docs/contributors/patching/maintain-ubuntu-desktop-software.md

```diff
@@ -0,0 +1,1045 @@
+    :::
+
+7. Commit the changes:
+
+    ```{terminal}
+    :copy:
+    :host:
+    :dir: gnome-control-center
+    :user:
+    :input: git commit -a
+    ```
+
+4. Push your changes to Salsa and track that branch:
+
+    ```{terminal}
+    :copy:
+    :host:
+    :dir: gnome-control-center
+    :user:
+    :input: git push -u origin ubuntu/noble
+
+    Total 0 (delta 0), reused 0 (delta 0)
+    To git+ssh://git@salsa.debian.org:gnome-team/gnome-control-center
+     * [new branch]          ubuntu/noble -> ubuntu/noble
+    Branch 'ubuntu/noble' set up to track remote branch 'ubuntu/noble' from 'origin'.
+    ```
+
+
+## Merge with Debian
+
+Do not forget to include up to the Debian version as a -v argument to dpkg-genchanges, so that it appears in the changes file too and is display.
+
+TO BE CONTINUED
+
+:::{admonition} TODO
+:class: attention
+We'll work on this with Nathan. It belongs under `dput` commands.
+:::
+
+
+## Build a package locally
+
+* Build a binary package: 
+
+    ```{terminal}
+    :copy:
+    :host:
+    :dir: gnome-control-center
+    :user:
+    :input: gbp buildpackage
+    ```
+
+* Build a source package: 
+
+    ```{terminal}
+    :copy:
+    :host:
+    :dir: gnome-control-center
+    :user:
+    :input: gbp buildpackage -S
+    ```
+
+With the proposed configuration, the artifacts all end up in the `../build-area` directory, including the tarball, which is reconstructed from the `pristine-tar` + `upstream/latest` branch. The build directory is then cleaned up.
+
+Useful tips in some potential cases:
+
+* Don't purge the `build-area` directory after building:
+
+    ```{terminal}
+    :copy:
+    :host:
+    :dir: gnome-control-center
+    :user:
+    :input: gbp buildpackage --git-no-purge
+    ```
+
+* Build current branch with local uncommitted modifications:
+
+    ```{terminal}
+    :copy:
+    :host:
+    :dir: gnome-control-center
+    :user:
+    :input: gbp buildpackage --git-ignore-new --git-export-dir="" 
+    ```
+
+
+## Release a new version
+
+:::{note}
+If this is a sponsored upload, the sponsor performs these steps.
+:::
+
+1. Generate the changelog for native packages. For details, see {ref}`desktop-git-update-the-changelog`.
+
+1. You might need to manually edit the `debian/changelog` file to properly group changes per people.
+
+1. Finalize the changelog. On the corresponding branch:
+
+    ```{terminal}
+    :copy:
+    :host:
+    :dir: gnome-control-center
+    :user:
+    :input: dch -r ""
+
+    :input: git commit debian/changelog -m "Finalize changelog"
+    ```
+
+1. Build the binary and source package:
+
+    ```{terminal}
+    :copy:
+    :host:
+    :dir: gnome-control-center
+    :user:
+    :input: gbp buildpackage --git-tag-only
+    
+    :input: gbp buildpackage -S
+    ```
+
+1. Upload the files to the Debian package upload queue:
+
+    ```{terminal}
+    :copy:
+    :host:
+    :dir: gnome-control-center
+    :user:
+    :input: dput ../build-area/….changes
```

Comment:
I would explicitly mention the file name

```suggestion
    :input: dput ../build-area/…_source.changes
```

This is the most delicate part, where we're actually pushing data to ubuntu, so it's important to be checked.

Also the .changes file should be manually reviewed before uploading